src/main/resources/db/seed/01_users_loadtest_students.sql

```
@@ -0,0 +1,37 @@
+
+SET SESSION cte_max_recursion_depth = 2000000;
+
+INSERT INTO users (
+    role, email, password, name, nickname, phone,
+    provider, provider_id, email_verified, is_locked,
+    bio, career, created_at, updated_at, deleted_at
+)
+WITH RECURSIVE seq (n) AS (
+    SELECT 1
+    UNION ALL
+    SELECT n + 1 FROM seq WHERE n < 1000000    -- 100만: 풀스캔+filesort 가 실제로 아파지는 규모
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)